plugins/discourse-presence/assets/javascripts/discourse/lib/presence.js.es6

```diff
@@ -102,7 +102,14 @@ const Presence = EmberObject.extend({
   },
 
   publish(state, whisper, postId, staffOnly) {
-    if (this.get("currentUser.hide_profile_and_presence")) {
+    var hiddenProfile = this.get(
```

Comment:
```suggestion
    let hiddenProfile = this.get(
```
var has weird scoping issues.

Reply:
Also this is a bit odd, checking for two different properties. I assume it's backwards compatibility? Maybe we can add a comment saying so, perhaps with a TODO to remove it in the future.